src/Components/Title/Title.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Row, Container } from "react-bootstrap";
+import "./Title.css";
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const Title = ({ text, image }) => {
+  return (
+    <Container style={{backgroundImage:`url(${image||"images/banner-img.jpg"})`}} fluid className="title-container vertical-center">
```

Comment:
No se deberian colocar estilos in-line. Fijate de crear un img y asignarle una clase y manejarlo asi para setear la imagen, o sea, por src de img y no por estilo